src/entities/EntitiesPackage.js

```diff
@@ -18,10 +18,9 @@ export default {
     config.addNode(BibliographicEntry)
     config.addNode(JournalArticle)
     config.addNode(ConferenceProceeding)
-    config.addNode(ClinicalTrial)
-    config.addNode(Preprint)
     config.addNode(DataPublication)
-    config.addNode(Periodical)
+    config.addNode(MagazineArticle)
+    config.addNode(NewspaperArticle)
     config.addNode(Patent)
```

Comment:
Do we need to increase the version number some lines above?
```js
version: '1.0.0',
```

Reply:
We can keep the version while in beta, but once Texture 1.0 is out we should start with versioning the schema, yes!